Extending/Dashboards/index.md

```diff
@@ -205,7 +205,7 @@ For reference, here is a list of the weighting values for the default Umbraco da
 
 After registering your dashboard, it will appear in the backoffice - however, it will have its dashboard alias [mycustomdashboard] wrapped in square brackets. This is because it is missing a language key. The language key allows people to provide a translation of the dashboard name in multilingual environments. To remove the square brackets - add a language key:
 
-If your dashboard is unique to your Umbraco installation, then you can modify the following application language file: `umbraco/config/lang/en-US.user.xml`. If the dashboard is to be released as an Umbraco package and shared with others to use in their own Umbraco installation, you will need to create a *lang* folder in your custom dashboard folder. You also need to create a package-specific language file:  `App_Plugins/Mycustomdashboard/lang/en-US.xml`.
+If your dashboard is unique to your Umbraco installation, then you can add or modify the relevant application language files: `config/lang/{language}.user.xml` (e.g. `config/lang/en-US.user.xml`). If the dashboard is to be released as an Umbraco package and shared with others to use in their own Umbraco installation, you will need to create a *lang* folder in your custom dashboard folder. You also need to create a package-specific language file:  `App_Plugins/Mycustomdashboard/lang/en-US.xml`.
```

Comment:
```suggestion
If your dashboard is unique to your installation, you can add or modify the relevant language files: `config/lang/{language}.user.xml` (e.g. `config/lang/en-US.user.xml`). If the dashboard is to be released as a Umbraco package, you will need to create a *lang* folder in your custom dashboard folder. You also need to create a package-specific language file:  `App_Plugins/Mycustomdashboard/lang/en-US.xml`.
```

Reply:
Hi Sofie, shouldn't that be "an Umbraco"? Otherwise, your suggestions look great. Thanks. I always change the bear minimum, rather than reviewing the whole thing!

Reply:
Well, I have Grammarly installed, and that told me it had to the "a Umbraco project" - I guess because it doesn't know the word Umbraco, and refers to "project" instead 🤔 

I just asked one of our native English speaking people here, and you're right! It should be **an Umbraco project** 😄

Reply:
I corrected it back 😄

Reply:
Yes, Grammarly always gets it wrong! It's very annoying for me, but I'm sure it's even worse for you!